src/sentry/static/sentry/app/components/events/sdkUpdates.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import SentryTypes from 'app/sentryTypes';
+
+import {t} from 'app/locale';
+
+class EventSdkUpdateSuggestion extends React.Component {
+  static propTypes = {
+    suggestion: PropTypes.object.isRequired,
+  };
+
+  getSuggestionTitle() {
+    const {suggestion} = this.props;
+    switch (suggestion.type) {
+      case 'updateSdk':
+        return <a href={suggestion.sdkUrl}>{t('update your SDK')}</a>;
```

Comment:
Might look better capitalized?

```suggestion
        return <a href={suggestion.sdkUrl}>{t('Update your SDK')}</a>;
```

Reply:
done

Reply:
Oops, didn't realize this was inline in a sentence, definitely should be lowercase, sorry

```suggestion
        return <a href={suggestion.sdkUrl}>{t('update your SDK')}</a>;
```